### PR TITLE
Fixed Sugarizer issue #414: Add a fullscreen button in ColorMyWorld activity

### PR DESCRIPTION
**Pull request by jontymagan**

### CODE REVIEW

File: activities/ColorMyWorld.activity/js/activity.js [hunk 1]
```diff
@@ -85,3 +85,16 @@ define([
...
 	});
 });
+		
+document.getElementById("fullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 0;
+			document.getElementById("canvas").style.top = "0px";
+			document.getElementById("unfullscreen-button").style.visibility = "visible";
+			resizeCanvas();
+		});
+document.getElementById("unfullscreen-button").addEventListener('click', function() {
```
Comment (FreddieN) [inline activities/ColorMyWorld.activity/js/activity.js]: Please can you ensure that the indentation of the document.getElementById.... follows the indentation style of the rest of the file.

File: activities/LastOneLoses.activity/js/activity.js [hunk 2]
```diff
@@ -485,3 +485,16 @@ define(["sugar-web/activity/activity", "sugar-web/graphics/radiobuttonsgroup", "
...
+			document.getElementById("canvas").style.top = "0px";
+			document.getElementById("unfullscreen-button").style.visibility = "visible";
+			resizeCanvas();
+		});
+		document.getElementById("unfullscreen-button").addEventListener('click', function() {
+			document.getElementById("main-toolbar").style.opacity = 1;
+			document.getElementById("canvas").style.top = "55px";
+			document.getElementById("unfullscreen-button").style.visibility = "hidden";
+			resizeCanvas();
+		});
```
Comment (FreddieN) [inline activities/LastOneLoses.activity/js/activity.js]: All of this should be within the define([.... etc.. of the activity same applies for the ColorMyWorld

File: activities/ColorMyWorld.activity/js/activity.js (hunk 1 above)
Comment (jontymagan) [reply activities/ColorMyWorld.activity/js/activity.js]: Done

File: activities/LastOneLoses.activity/js/activity.js (hunk 2 above)
Comment (jontymagan) [reply activities/LastOneLoses.activity/js/activity.js]: Done

**Comment by jontymagan** [comment]
Added Fullscreen and Unfull Screen button

**Pull request closed without merging**